guides/office/kitchen.md

```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have multiple thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
+
+ - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
```

Comment:
Is the cupboard trick acceptable?

Reply:
I think batch grind at beginning of day more acceptable.